### PR TITLE
Second challenge of Css challenges with image

### PR DESCRIPTION
**Pull request by susird**
This is the pr of **CSS Push Button**  form css challenges :sparkles:
image

### CODE REVIEW

**Review by josepplloo — CHANGES_REQUESTED** [review]
Use margin to avoid repeating yourself in margin-right margin-left and in general we try tu use position relative in the container and absolute in the elements inside the container

File: challenge_2/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <title>Document</title>
+</head>
+<body>
+  <div class="container">
+    <div class="boxInside"></div>
+    <div class="bigCircle"></div>
+    <div class="smallCircle"></div>
+  </div>
+</body>
+</html>
```
Comment (anyruizd) [inline challenge_2/index.html]: Remember to add a blank line at the end of the file.
```suggestion
</html>

```

File: challenge_2/style.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.smallCircle {
+  background: #EEB850;
+  border-radius: 100%;
+  height: 50px;
+  margin-left: 175px;
+  margin-top: 125px;
+  position: relative;
+  width: 50px;
+}
```
Comment (anyruizd) [inline challenge_2/style.css]: Same here
```suggestion
}

```

File: challenge_2/style.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  height: 150px;
+  margin-left: 50px;
+  margin-top: 75px;
+  position: absolute;
+  width: 300px;
+}
+
+.bigCircle {
+  border: 60px solid #6592CF;
+  border-radius: 100%;
```
Comment (anyruizd) [inline challenge_2/style.css]: It's not necessary to use 100%, if you want to accomplish a full circle, with `border-radius: 50%` would be enough 😉

**Review by josepplloo — CHANGES_REQUESTED** [review]
Why `.container` have position absolute?

File: challenge_2/index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  <title>Document</title>
+</head>
+
+<body>
+  <div class="container">
+    <div class="boxInside">
+      <div class="bigCircle">
+        <div class="smallCircle"></div>
+      </div> 
+
```
Comment (josepplloo) [inline challenge_2/index.html]: fix those spaces

**Review by josepplloo — CHANGES_REQUESTED** [review]

**Review by josepplloo — APPROVED** [review]
lgtm 🚀

File: challenge_2/style.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  border: 50px solid #6592CF;
+  border-radius: 50%;
+  height: 250px;
+  padding: 50px;
+  width: 250px;
+}
+
+.smallCircle {
+  background: #EEB850;
+  border-radius: 100%;
```
Comment (anyruizd) [inline challenge_2/style.css]: 😆

**Review by anyruizd — APPROVED** [review]
just one small comment

**Pull request merged**